# goal
geometry_msgs/PoseStamped goal_pose
string target_map
---
# result
bool success
string message
---
# feedback
float32 percent_complete
